src/main/resources/application.yaml

```diff
@@ -24,6 +24,9 @@ spring:
       - 'classpath:db/changelog/v.0.4/sql-import/import-user-internship.sql'
       - 'classpath:db/changelog/v.0.4/sql-import/import-user-hash-password.sql'
       - 'classpath:db/changelog/v.0.3/sql-import/import-time-for-call.sql'
+      - 'classpath:db/changelog/v.0.5/sql-import/location.sql'
```

Comment:
strange order of changes: 

1. v.0.4
2. v.0.3
3. v.0.5

Reply:
Can I Leave now this variant ? Because I need time to think the orders appended columns to order them ideally 
![image](https://user-images.githubusercontent.com/61596282/116532309-c5d6cd80-a8e8-11eb-8db0-d9d104c9f8b5.png)

Reply:
Correct please in next PR